proto/users/users.proto

```diff
@@ -44,7 +57,5 @@ message GetApproverInfoRequest {
 }
 
 message GetApproverInfoResponse {
-    bool canApprove = 1;
-    string unit = 2;
-    repeated string ranks = 3;
+    ApproverInfo approverInfo =  1;
```

Comment:
Has the return value been tested with Rabaz? and how?

Reply:
tested in kubernetes with rabaz.